src/js/commands/italic.js

```diff
@@ -14,10 +14,21 @@ export default class ItalicCommand extends TextFormatCommand {
     this.markup = builder.createMarkup('em');
   }
   exec() {
-    this.editor.applyMarkupToSelection(this.markup);
+    let markerRange = this.editor.cursor.offsets;
+    if (!markerRange.leftRenderNode || !markerRange.rightRenderNode) {
+      return;
+    }
```

Comment:
Ah. I was able to create a failure here in chrome with some wacky selection drags. We should look at it together, I'll make a ticket.